### ISSUE TITLE
Please create a config for HP Pavilion 15-dk1xxx

### ISSUE THREAD

**Issue by YousefAhmed2812** [UraniumDonut/nbfc-revive#77]
i tried creating my own config but the instructions aren't very clear and i don't understand most of it

**Comment by TiagoRibeiro25** [comment]
I have a config for that, but it only has for the cpu fan. When enabling the config, the gpu fan goes max speed

**Comment by TiagoRibeiro25** [comment]
[https://www79.zippyshare.com/v/OGq6DGSl/file.html](url)
If you manage to find the values for the gpu fan, please share :)

**Comment by YousefAhmed2812** [comment]
will do, thank you so much

**Comment by YousefAhmed2812** [comment]
yeah that happened to me i reinstalled nbfc and kept trying and i got it to work but was unable to stop it, the only way i was able to stop it was to stop the service from starting up with windows, and it was very inconsistent some times it worked and sometimes it didn't.

**Comment by Fertaz** [comment]
> yeah that happened to me i reinstalled nbfc and kept trying and i got it to work but was unable to stop it, the only way i was able to stop it was to stop the service from starting up with windows, and it was very inconsistent some times it worked and sometimes it didn't.

I found the fix ! You have to change a dll file from plugins then restart your laptop, it worked for me

**Comment by Fertaz** [comment]
https://github.com/UraniumDonut/nbfc-revive/issues/27#issuecomment-967684864 - this is the dll file that you need to change

**Comment by YousefAhmed2812** [comment]
@TiagoRibeiro25 can you please reupload the config file, the old one expired in Zippyshare

**Comment by TiagoRibeiro25** [comment]
> @TiagoRibeiro25 can you please reupload the config file, the old one expired in Zippyshare

https://www.mediafire.com/file/22s30kjgt5mz5m7/HP_Pavilion_Gaming_Laptop_15-dk1xxx.xml/file

**Comment by YousefAhmed2812** [comment]
> > @TiagoRibeiro25 can you please reupload the config file, the old one expired in Zippyshare
> 
> https://www.mediafire.com/file/22s30kjgt5mz5m7/HP_Pavilion_Gaming_Laptop_15-dk1xxx.xml/file
> 
> The gpu reading is 15, but for some reason I can only change it to 100% speed or 0% speed

The gpu reading isn't in this config right? and just for reference what kind of temperatures are you getting for the cpu and gpu under load.

**Comment by TiagoRibeiro25** [comment]
> > > @TiagoRibeiro25 can you please reupload the config file, the old one expired in Zippyshare
> > 
> > 
> > [mediafire.com/file/22s30kjgt5mz5m7/HP_Pavilion_Gaming_Laptop_15-dk1xxx.xml/file](https://www.mediafire.com/file/22s30kjgt5mz5m7/HP_Pavilion_Gaming_Laptop_15-dk1xxx.xml/file)
> > The gpu reading is 15, but for some reason I can only change it to 100% speed or 0% speed
> 
> The gpu reading isn't in this config right? and just for reference what kind of temperatures are you getting for the cpu and gpu under load.

It's, when you enable the config file (change to enable) the gpu fan will go full speed. If you don't want that, you can just delete the only configuration in the “register write configuration” section.
Under load, I usually don't get more than 80ºC on the cpu (keep in mind, I undervolted my intel cpu).

**Comment by YousefAhmed2812** [comment]
i have the i5-10300h  so unfortunately, i can't undervolt it and my cpu temps are in the 90s i tried repasting and got one of the best thermal pastes available and the temps are still in the 90s.

**Comment by TiagoRibeiro25** [comment]
> i have the i5-10300h  so unfortunately, i can't undervolt it and my cpu temps are in the 90s i tried repasting and got one of the best thermal pastes available and the temps are still in the 90s.

I have the i5-10300h as well. Undervolt is locked by default, so I unlocked following the first part of this guide: https://www.reddit.com/r/Hewlett_Packard/comments/o1zxuv/howto_unlock_undervolting_change_timings_on_hp/?sort=new

**Comment by YousefAhmed2812** [comment]
Thank you so much for that i have been looking for a way to unlock it and be able to undervolt i tried it and it worked thank you so much.

**Comment by TiagoRibeiro25** [comment]
> Thank you so much for that i have been looking for a way to unlock it and be able to undervolt i tried it and it worked thank you so much. also how much have you been able to undervolt.

Right now, I have -135mv, but it doesn't mean it will be stable for you. I'm pretty sure you can get at least -75mv, just start there and keep raising until you find errors or blue screen in the stress test.

**Comment by YousefAhmed2812** [comment]
Will do, quick question do you also undervolt the cache or just the core. I was able to do -150mv on the core and it seems to be stable

**Comment by TiagoRibeiro25** [comment]
> Will do, quick question do you also undervolt the cache or just the core. I was able to do -150mv on the core and it seems to be stable

Both

**Comment by Sayyid01** [comment]
> > i have the i5-10300h  so unfortunately, i can't undervolt it and my cpu temps are in the 90s i tried repasting and got one of the best thermal pastes available and the temps are still in the 90s.
> 
> I have the i5-10300h and I had temps in 90s as well before. Undervolt is locked by default, so I unlocked it by following the first part of this guide (part 1, up to the step 5): https://www.reddit.com/r/Hewlett_Packard/comments/o1zxuv/howto_unlock_undervolting_change_timings_on_hp/?sort=new

This does not apply to HP Pavilion Gaming Laptop - 15-dk1064tx, the UEFI is protected, I've tried it multiple times. the only thing that can help with the thermal issue is this, but we need to find out about the EC (Embedded Controller) in this device.

**Comment by TiagoRibeiro25** [comment]
> > > i have the i5-10300h  so unfortunately, i can't undervolt it and my cpu temps are in the 90s i tried repasting and got one of the best thermal pastes available and the temps are still in the 90s.
> > 
> > 
> > I have the i5-10300h and I had temps in 90s as well before. Undervolt is locked by default, so I unlocked it by following the first part of this guide (part 1, up to the step 5): https://www.reddit.com/r/Hewlett_Packard/comments/o1zxuv/howto_unlock_undervolting_change_timings_on_hp/?sort=new
> 
> This does not apply to HP Pavilion Gaming Laptop - 15-dk1064tx, the UEFI is protected, I've tried it multiple times. the only thing that can help with the thermal issue is this, but we need to find out about the EC (Embedded Controller) in this device.

Few months ago, when I used to use Windows 11, windows updated the bios automaticly through windows updates (I thought windows couldn't update a BIOS...). After that, the method to unlock undervolting (the one mentioned in that reddit post) doesn't work anymore and for some reason it doesn't let me roll back to a previous BIOS version. So ye, currently I also have undervolt blocked and I haven't found a way to unlock it. Using this software to increase the fans speed does help, but undervolting was the method that used to most help me when it comes to decreasing the cpu temps. 😥

**Comment by Sayyid01** [comment]
@TiagoRibeiro25 do you knew how to analyze this laptop DSDT and find out about the EC's Register? I want to create a fully working config file, but not able to access the DSDT, let alone the EC's. Already try 2 software recomended [here](https://github.com/hirschmann/nbfc/wiki/How-to-create-a-NBFC-config), but nothing work as it intended to be.

**Comment by TiagoRibeiro25** [comment]
> @TiagoRibeiro25 do you knew how to analyze this laptop DSDT and find out about the EC's Register? I want to create a fully working config file, but not able to access the DSDT, let alone the EC's. Already try 2 software recomended [here](https://github.com/hirschmann/nbfc/wiki/How-to-create-a-NBFC-config), but nothing work as it intended to be.

I do not. I just use the "HP Compaq 15-s103tx" config file that just works. The only downside of that config for me is that it sets the gpu fan to max speed. So I only use nbfc when I'm going to play demanding games.

**Comment by Chandrakanthck** [comment]
it works with " HP Compaq 15-s103tx" config file " 
specs: i5-10300H
HP PAVILION GAMING 15 dk1xxxxx series laptop

**Comment by Kyr4l** [comment]
> it works with " HP Compaq 15-s103tx" config file " specs: i5-10300H HP PAVILION GAMING 15 dk1xxxxx series laptop

it pushes the gpu fan speed to 100%, unusable in most cases

**Comment by Chandrakanthck** [comment]
what is your laptop model? adn specs? Mine workings good. it is different for differnt laptop/cpu/motherboard
